t/14-isotovideo.t

```diff
@@ -145,6 +143,16 @@ subtest 'isotovideo with wheels' => sub {
     $specfile->remove;
     is(checkout_wheels("$wheels_dir"), 1, 'no wheels');
     is(scalar @repos, 5, 'git never called');
+
+    # also verify that isotovideo invokes the wheel code correctly
+    $bmwqemu::vars{CASEDIR} = "$data_dir/tests";
+    $specfile->spurt("version: v0.1\nwheels: [copy/writer]");
+    path($pool_dir, 'writer', 'lib', 'Copy', 'Writer')->make_path->child('Content.pm')->spurt("package Copy::Writer::Content; sub write {}; 1");
+    path($pool_dir, 'writer', 'tests', 'pen')->make_path->child('ink.pm')->spurt("use Mojo::Base 'basetest', -signatures; use Copy::Writer::Content 'write'; sub run (\$) {}; 1");
```

Comment:
`write` is not a good function name because it's a builtin function. And it's also not exported yet.
For making sure the `run` function is actually called I guess I would make a `diag` call and check for that in the log.

Reply:
For the record, with exporter and exit_after_schedule the test works just fine and is a bit faster - it's just unfortunate that without it perl completely ignores exports, so this will need to be documented clearly for users